assets/src/edit-story/app/media/utils/preloadImage.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const preloadImage = (src) => {
```

Comment:
Pls add jsdoc.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/1102/commits/2dec80bbee64a94606316a3c4b08c4d97c9ac7bd